src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -43,6 +43,7 @@ class SecurityExtension extends Extension
     private $factories = array();
     private $userProviderFactories = array();
     private $expressionLanguage;
+    private $logoutOnUserChangeByContextKey = array();
```

Comment:
@chalasr should I add some deprecation info here? Did not see that for member attributes elsewhere?

Reply:
Let's let it as is and keep it in mind, note for mergers: this prop should be removed when merging in 4.0 (I can take care of the merge)